packages/network/src/identifiers.ts

```diff
@@ -13,15 +13,15 @@ export interface Location {
     city?: string
 }
 
-export interface StreamStatus {
-    id: string
+export interface StreamPartStatus {
+    id: StreamID
```

Comment:
I'm not sure we can ensure this type in the use case where the tracker receives a `StatusMessage`.

Reply:
When we build a validation (#255), we can validate this field